packages/core/src/components/block/block.tsx

```diff
@@ -36,13 +62,13 @@ export class TdsBlock {
         : '';
 
     return (
-      <div
+      <section
```

Comment:
A small suggestion that might be more applicable to other components, but I'll leave it here anyway.. 

There might be a scenario where a user would like to control what element i rendered here, sp they can create proper semantic HTML. So my suggestion would be to introduce an `as` prop, where the user could do: 

```jsx
<tds-block as="div">
   // content
</tds-block>
```

Just an idea, keep up the great work! 🚚

Reply:
Good point! Added a prop called componentTag 🤘